tests/nlu/extractors/test_extractor.py

```diff
@@ -96,13 +96,14 @@
         (
             "Amsterdam Berlin and London",
```

Comment:
Can you please add some more test cases like this? E.g. "San Francisco Amsterdam, London" or even something with three tokens? Thanks.

Reply:
Added more test cases to cover 1-3 tokens.